editor/src/core/model/project-file-helper-hooks.ts

```diff
@@ -26,3 +29,55 @@ export function useReParseOpenProjectFile(): () => void {
     }
   }, [refEditorState, dispatch])
 }
+
+function spliceCode(
+  originalCode: string,
+  originalStringData: SteganoTextData,
+  updatedString: string,
+): string {
+  const currentString = originalCode.slice(
```

Comment:
Nit: I think `currentString` is a bit confusing when used with `originalCode` and `originalStringData`. Can we rename it to `originalString` please

Reply:
renamed in https://github.com/concrete-utopia/utopia/pull/4480/commits/5eac2a988c11e657273664a00a44a31faad7359f